integration_tests/auction_test.go

```diff
@@ -117,6 +121,13 @@ func (s *IntegrationTestSuite) TestAuctionModule() {
 		s.T().Logf("Auction module token balances before bids %v", balanceRes.Balances)
 
 		initialOrchBalanceRes, err := bankQueryClient.AllBalances(context.Background(), &banktypes.QueryAllBalancesRequest{Address: orch0Address})
+		var initialOrchSomm sdk.Int
+		// initialOrchGravity := sdk.ZeroInt()
+		for _, balance := range initialOrchBalanceRes.Balances {
+			if balance.Denom == "usomm" {
```

Comment:
We're pulling the string `usomm` from `BaseCoinUnit` elsewhere, should do it here too.

Reply:
Actually I think we should change references to `BaseCoinUnit` to `testDenom` in the tests